### PR TITLE
fix(fxLayoutAlign): add space-between and space-around options

### LINKED ISSUE #841: fxLayoutAlign doesn't work with space-around and space-between (cross-axis)
# Bug Report

#### What is the expected behavior?
fxLayoutAlign should allow to use all the flex options for the cross axis.

#### What is the current behavior?
fxLayoutAlign doesn't work with space-around and space-between in the cross-axis. When used, it defaults to 'stretch'.

In the source code (_buildCSS in layout-align.ts), this options are not covered:
captura de pantalla 2018-09-14 a las 9 11 08

#### What are the steps to reproduce?
https://stackblitz.com/edit/flex-layout-issue-2?file=app%2Fapp.component.html
captura de pantalla 2018-09-14 a las 9 08 07

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
All.

#### Is there anything else we should know?
Thank you for your work, great concept and library!

### PR DESCRIPTION
**Pull request by aleixsuau**
The actual implementation doesn't allow the 'space-between' and 'space-around' flex
align-content options (cross-axis). fxLayoutAlign should cover all of them.

Closes #841

### CODE REVIEW

File: src/lib/flex/layout-align/layout-align.spec.ts [hunk 1]
```diff
@@ -5,22 +5,16 @@
...
-} from '@angular/flex-layout/core';
-
-import {FlexLayoutModule} from '../../module';
-import {extendObject} from '../../utils/object-extend';
-import {customMatchers} from '../../utils/testing/custom-matchers';
-import {makeCreateTestComponent, expectNativeEl} from '../../utils/testing/helpers';
+import { Platform } from '@angular/cdk/platform';
+import { CommonModule } from '@angular/common';
+import { Component, OnInit } from '@angular/core';
+import { ComponentFixture, inject, TestBed } from '@angular/core/testing';
```
Comment (CaerusKaru) [inline src/lib/flex/layout-align/layout-align.spec.ts]: Please don't refactor our stylings

File: src/lib/flex/layout-align/layout-align.ts [hunk 2]
```diff
@@ -5,23 +5,14 @@
...
-  SimpleChanges,
-  Self,
-} from '@angular/core';
-import {BaseDirective, MediaChange, MediaMonitor, StyleUtils} from '@angular/flex-layout/core';
-import {Subscription} from 'rxjs';
-
-import {extendObject} from '../../utils/object-extend';
-import {Layout, LayoutDirective} from '../layout/layout';
-import {LAYOUT_VALUES, isFlowHorizontal} from '../../utils/layout-validator';
+import { Directive, ElementRef, Input, OnChanges, OnDestroy, OnInit, Optional, Self, SimpleChanges } from '@angular/core';
```
Comment (CaerusKaru) [inline src/lib/flex/layout-align/layout-align.ts]: Same here with styling

File: src/lib/flex/layout-align/layout-align.spec.ts (hunk 1 above)
Comment (aleixsuau) [reply src/lib/flex/layout-align/layout-align.spec.ts]: Sorry, my VSCode refators stylings automatically.
Fixed and "git rebase master -i" and then "git push -f".

**Comment by CaerusKaru** [comment]
Marking as blocked until I discuss with @ThomasBurleson, but otherwise looks good!

**Pull request merged**